xml/System.IO/FileSystemInfo.xml

```diff
@@ -1291,7 +1291,17 @@ On Unix platforms that do not support creation or birth time, this property retu
       <Docs>
         <summary>Returns the original path. Use the <see cref="P:System.IO.FileSystemInfo.FullName" /> or <see cref="P:System.IO.FileSystemInfo.Name" /> properties for the full path or file/directory name.</summary>
         <returns>A string with the original path.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+[!IMPORTANT]
+
+> The string returned by the <xref:System.IO.FileSystemInfo.ToString%2A> method represents path that was passed to the constructor. When you create a <xref:System.IO.FileSystemInfo> object using the constructors, the <xref:System.IO.FileSystemInfo.ToString%2A> method returns the fully qualified path. However, there are cases where the string returned by the <xref:System.IO.FileSystemInfo.ToString%2A> method does not represent the fully qualified path. To ensure consistent behavior and explicit intent, call `ToString` on the <xref:System.IO.FileSystemInfo.Name> or <xref:System.IO.FileSystemInfo.FullName> of the current <xref:System.IO.FileSystemInfo> instance.
```

Comment:
```suggestion
> The string returned by the <xref:System.IO.FileSystemInfo.ToString%2A> method represents the path that was passed to the constructor of the class implementing <xref:System.Io.FileSystemInfo>. Instead of calling the `ToString` method, you should retrieve the value of either of the following properties, depending on your intent:
- <xref:System.IO.FileSystemInfo.Name>, to get the name of the file or directory, without any parent path information.
- <xref:System.IO.FileSystemInfo.FullName>, to get the fully qualified path of the file or directory.
```